halo2_proofs/src/poly/kzg/multiopen/shplonk.rs

```diff
@@ -118,15 +118,21 @@ where
         };
     }
 
+    // Note: due to storing the execution engine in the CommitmentScheme
+    // it cannot be Send+Sync (some execution engines might use thread-local storage)
+    // TODO: refactor so that commitment do not have to carry the execurtion engine.
     let rotation_sets = rotation_set_commitment_map
-        .into_par_iter()
+        // .into_par_iter()
+        .into_iter()
         .map(|(rotations, commitments)| {
             let rotations_vec = rotations.iter().collect::<Vec<_>>();
             let commitments: Vec<Commitment<F, Q::Commitment>> = commitments
-                .into_par_iter()
+                // .into_par_iter()
+                .into_iter()
                 .map(|commitment| {
                     let evals: Vec<F> = rotations_vec
-                        .par_iter()
+                        // .par_iter()
+                        .iter()
```

Comment:
This requires Send + Sync but having ZalEngine being part of Commitment prevents that